generators/entity-server/templates/dotnetcore/src/Project/Service/Mapper/Profile.cs.ejs

```diff
@@ -0,0 +1,38 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+<%_ let importJsonIgnore = false;
+const hasManyToMany = entityClassHasManyToMany;
+const entityTableName = snakeCasedEntityClass;
+const entityClassName = pascalizedEntityClass;
+const entityVariableName = camelCasedEntityClass; _%>
+using AutoMapper;
+using <%= namespace %>.Models;
+using <%= namespace %>.Service.Dto;
+
+namespace <%= namespace %>.Service.Mapper {
+
+    public class <%= entityClassName %>Profile : Profile
```

Comment:
normally it's one profile, can you create one profile and add all mapping to this ?

Reply:
Yes ! I agree with you @nicolas63 we can create one profile as follow.  @ivanmonteiro Don't forget to add mapper for User

```
public class AutoMapperProfile : Profile {
        public AutoMapperProfile()
        {
            CreateMap<User, UserDto>()
                .ForMember(userDto => userDto.Roles, opt => opt.MapFrom(user => user.UserRoles.Select(iur => iur.Role.Name).ToHashSet()))
            .ReverseMap()
                .ForPath(user => user.UserRoles, opt => opt.MapFrom(userDto => userDto.Roles.Select(role => new UserRole { Role = new Role { Name = role }, UserId = userDto.Id }).ToHashSet()));

            CreateMap<<%= entityClassName %>, <%= entityClassName %>Dto>().ReverseMap();
        }
    }
```

Reply:
Sure, I can move it all to one file and change the startup/AddAutoMapper

Reply:
You can let `@this.AddAutoMapper(typeof(Startup));`
The method AddAutoMapper scan all assembly who the Startup is located